app/javascript/packs/personal-key-page-controller.js

```diff
@@ -25,74 +24,15 @@ function scrapePersonalKey() {
 
 const personalKey = scrapePersonalKey();
 
-// The following methods are strictly fallbacks for IE < 11. There is limited
```

Comment:
RIP IE 11

Reply:
Just to clarify, native form validation would work fine in IE11 (even IE10 apparently). The only thing this would have been supporting is IE9 and older.

https://caniuse.com/form-validation